frontend/components/card/search-result/CardSearchResult.vue

```diff
@@ -27,6 +27,12 @@
     >
       <CardSearchResultUser :user="user" :isPrivate="isPrivate" />
     </div>
+    <div
```

Comment:
I'm going to take this part out of here as well. This component is supposed to be `CardDiscussionEntry` based on the issue, which I get was confusing from the designs. It's styled like a search result, but we should keep all of this and what you're working on in #460 in a `discussion` directory as that to me seems like a more fitting categorization 😊